modules/registry-operator-distribution-across-availability-zones.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/configuring-registry-operator.adoc
+
+
+[id="registry-operator-distribution-across-availability-zones_{context}"]
+= Image Registry Operator distribution across availability zones
+
+The default configuration of the Image Registry Operator spreads image registry pods across topology zones to prevent delayed recovery times in case of a complete zone failure where all pods are impacted. 
+
+The Image Registry Operator defaults to:
+[source,yaml]
+----
+  topologySpreadConstraints:
+  - maxSkew: 1
+    topologyKey: topology.kubernetes.io/zone
+    whenUnsatisfiable: DoNotSchedule
+    labelSelector:
+      matchLabels:
+        docker-registry: default
+  - maxSkew: 1
+    topologyKey: kubernetes.io/hostname
+    whenUnsatisfiable: DoNotSchedule
+    labelSelector:
+      matchLabels:
+        docker-registry: default
+  - maxSkew: 1
+    labelSelector:
+      matchLabels:
+        docker-registry: default
+    topologyKey: node-role.kubernetes.io/worker
+    whenUnsatisfiable: DoNotSchedule
+----
+
+A cluster administrator can override the default `topologySpreadConstraints` by configuring `ImageRegistry` Spec. In that case, only the constraints you provided apply.
```

Comment:
Better to change s/`ImageRegistry`/`configs.imageregistry.operator.openshift.io/cluster`

Reply:
Could we add this bug scenario, https://bugzilla.redhat.com/show_bug.cgi?id=2067060#c3  ?
When there is no zone label setting, the topologySpreadConstraints will be default to
```
  - maxSkew: 1 
    topologyKey: kubernetes.io/hostname 
    whenUnsatisfiable: DoNotSchedule
    labelSelector:
      matchLabels:
        docker-registry: default
  - maxSkew: 1 
    labelSelector:
      matchLabels:
        docker-registry: default
    topologyKey: node-role.kubernetes.io/worker
    whenUnsatisfiable: DoNotSchedule
```